README.md

```diff
@@ -227,6 +227,36 @@ Software is installed in miniconda environments.  You can use "[conda list](http
 ### How can I contribute to existing Open-Source Fornax notebook tutorials?
   * open issue or PR on Fornax Github [repo](https://github.com/nasa-fornax/fornax-demo-notebooks)
 
+## Using Fornax Software Environments Outside Fornax
+The Fornax software environments are available as docker container images. To use them on your computer, you will need to install
+[docker](https://docs.docker.com/engine/install/). The following are the steps needed to run the notebooks locally using the Fornax images:
+
+- Start in a empty working direcotry that will be used used for the notebooks and any data generated during the analysis:
+```sh
+mkdir work
+cd work
+```
+- Clone the notebook repo: `git clone https://github.com/nasa-fornax/fornax-demo-notebooks.git`.
+- Download the relevant Fornax image. In this case, we are using the `tractor-stable` image. `docker pull public.ecr.aws/f6e2z3b0/fornax-images:tractor-stable`.
```

Comment:
Is the name of this image 'tractor-stable' going to always be the same name in perpetuity, or will the names change?  If the names will change, or if we want users to be able to see multiple different environments, then can you teach us users how to figure out the full list of available images?  as well as corresponding addresses for those images (ie., is the 'f6e2...' the same for all images?